src/utils.ts

```diff
@@ -140,14 +141,58 @@ export function toExports (imports: Import[], fileDir?: string) {
     })
     .join('\n')
 }
+export function extractJSDoc(modulePath: string, functionName: string) {
+  try{
+    
+    //Relative paths
+    if(modulePath.indexOf("../../")!==-1){
+      modulePath=modulePath.slice(6)
+      modulePath=existsSync(modulePath+".d.ts") ? modulePath+".d.ts" : modulePath+"/index.d.ts"
+    }
+    
+    //NPM packages (including scoped ones) (with handling for i18n, tested on Nuxt only, needs proper testing on other tools)
+    if(modulePath.indexOf("/")===-1 || modulePath.indexOf("@")!==-1){
+      modulePath=(modulePath.indexOf("node_modules")===-1?"node_modules/":"")+modulePath+(modulePath.indexOf("i18n")===-1? "/dist/"+( modulePath==="vue" ? modulePath+".d.ts": "index.d.ts"): '')
+    }
+    
+    //Absolute paths
+    if(modulePath.indexOf("/node_modules")!==-1){
+      modulePath=modulePath.slice(modulePath.indexOf("/node_modules")+1)
+      modulePath=existsSync(modulePath+".d.ts") ? modulePath+".d.ts" : modulePath+"/index.d.ts"
+    }
+    
+    //Vue-router (non-Nuxt locked)
+    if(modulePath.indexOf("vue-router")!==-1){
+      modulePath=existsSync(".nuxt") ? ".nuxt/vue-router.d.ts" : "node_modules/vue-router/dist/vue-router.d.ts"
+    }
```

Comment:
We should not hard-code them. And `unimport` should not rely on `typescript` directly. I would suggest making this more modular and customizable.

Reply:
@antfu I was thinking of making it less hard coded, the issue is the auto importing process can differ (a lot), where sometimes its absolute paths, sometimes it's relative paths, sometimes it's plain NPM packages. There are so many different variations of imports I encountered originally when testing this on Nuxt, so i had to account for every one. Would appreciate any idea to make it more dynamic! Regarding relying on typescript, I'll have to look into other ways of extracting the JSDoc, because all I found so far was just this method with the source file, would appreciate any suggestions here as well!